src/selectors/blockchain.js

```diff
@@ -23,3 +28,27 @@ export const getTokenAmount = (state, tokenAddress) => {
 }
 
 export const getTokenSymbol = (state, tokenAddress) => state.blockchain.getIn(['tokenSymbols', tokenAddress])
+
+/**
+ * @param {*} state - redux state
+ * @returns {object} collateralToken - with Symbol, Amount in ETH and Address
+ */
+export const getCollateralTokenAmount = (state) => {
```

Comment:
Isn't the name misleading? It returns not only amount, but symbol, address, icon too :) Maybe `getCollateralTokenInfo` would be a better name